README.md

```diff
@@ -675,6 +675,17 @@ updated to reflect that change. To fix this, redeploy the lambdas (`make
 package`) in the affected deployment.
 
 
+##  Cannot update googleSubjectId with SAM
+
+If you have destroyed your deployment and rebuild it, its possible that SAM will
+not allow the Google service account to not be registered due to the account id
+being accociated to a unique-id/googleSubjectId. To get around this increment 
+the current value of $AZUL_DEPLOYMENT_INCARNATION in the deployments
+environment.local.py file. Let a TDR Authorization admin know of the change of
+the service accound id via Slack. After the authorization is complete perform
+another redeploy to update infrastructure resources.
```

Comment:
```suggestion
If you have destroyed your deployment and are rebuilding it, it's possible that SAM will
not allow the Google service account to be registered again because the account
name is the same in the current and previous incarnation of the deployment, while the service account's subject ID is be different. SAM does not support this.

[Describe symptom here]

To get around this, increment 
the current value of `AZUL_DEPLOYMENT_INCARNATION` in the deployment's
`environment.py` file.

Let a TDR admin know of the change of
the service account name via Slack [how would I know what the new name is?]. After the authorization is complete run [what, be specific] to update infrastructure resources.
```

This is riddled with grammar, spelling and factual inaccuracies. Back to the drawing board.